script/Arcueid/Day 7/07_00_ARC07_1B.txt

```diff
@@ -393,37 +445,46 @@ Thanks to that, this morning is going to be even... more... energetic?
 [sha:3f0ba8736934b0c1ce948ec95157ad4e44a2c1c1]{
 -- Page 71, Offset 16751.
 -- 「寝こんでないし、貧弱でもない。
+// This correct for 貧弱でもない?
```

Comment:
I think he's saying like "I wasn't asleep all weekend, I'm not some feeble loser" or something like he's defending his health.
> I didn't stay in bed, and I'm not feeble either.